lib/common/output_xml.c

```diff
@@ -176,10 +201,14 @@ xml_finish(pcmk__output_t *out, crm_exit_t exit_status, bool print, void **copy_
     /* If root is NULL, xml_init failed and we are being called from pcmk__output_free
      * in the pcmk__output_new path.
      */
-    if (priv == NULL || priv->root == NULL) {
+    if (priv == NULL) {
```

Comment:
The comment above is no longer associated here, but still might matter as to what we do here